src/util.js

```diff
@@ -202,6 +208,12 @@ module.exports = {
    * @return {String} String representation of the array
    */
   Uint8Array2str: function (bin) {
+
+    // Uncomment for debugging
+    if(!Uint8Array.prototype.isPrototypeOf(bin)) {
+      throw new Error('Uint8Array2str: Data must be in the form of a Uint8Array');
+    }
+
```

Comment:
Yeah, it was for tracking down places where I hadn't done the string to Uint8Array conversion. I never looked into whether or not the performance hit was noticeable--my guess is probably not, so maybe we should uncomment and leave it in. It'll make type errors easy to find.